internal/llm/provider/gemini.go

```diff
@@ -327,7 +327,7 @@ func (g *geminiClient) stream(ctx context.Context, messages []message.Message, t
 						return
 					}
 					if retry {
-						slog.Warn("Retrying due to rate limit", "attempt", attempts, "max_retries", maxRetries)
+						slog.Warn("Retrying due to rate limit", "attempt", attempts, "max_retries", maxRetries, "after_ms", after)
```

Comment:
Since the retry time gradually increases, I think it would be helpful to output it to the log. What do you think about this?